contracts/Airdrop.sol

```diff
@@ -70,37 +74,48 @@ contract Airdrop is Ownable {
         info = airDrops[_airdropID];
```

Comment:
Remove require check.
Check if another contract can access this function without difficulties.

Reply:
Fixed in [fa35427](https://github.com/MisterDefender/Airdrop-ERC-20/pull/2/commits/fa354274c837d5e19adbbf4e81cd8a5444bb1c20)